examples/random/src/main.cpp

```diff
@@ -95,7 +95,7 @@ vector_example(sycl::queue& queue, std::uint32_t seed, std::vector<RealType>& x)
                         auto res = distr(engine);
 
                         // Store results from res to VecSize * offset position of x_acc
-                        res.store(idx.get_linear_id(), x_acc.get_pointer());
+                        res.store(idx.get_linear_id(), x_acc);
```

Comment:
A more straightforward solution would be to pass `x_acc.template get_multi_ptr<sycl::access::decorate::yes>()`. But `multi_ptr` can be constructed using a buffer accessor, so we can pass the accessor, which looks simpler.